backend/src/main/java/com/google/step/interfaces/ClientResponse.java

```diff
@@ -0,0 +1,11 @@
+package com.google.step.interfaces;
+
+/**
+ * Represents responses sent back to the client.
+ */
+public interface ClientResponse {
+    /**
+     * Returns this response as a String in JSON format.
+     */
+    public String getJson();
```

Comment:
would a better name potentially be toJson() similar to a toString() method?
getJson should be fine, but getJson sort of reminds me of a getter method where json is an instance var where toJson feels like a transformation like toString does.
Also, like the idea of making this an interface 👍

Reply:
Yeah, I agree the naming is weird on this one. Will change.